.github/workflows/release_note.py

```diff
@@ -210,36 +212,95 @@ def generate_package_update_section(version):
                     output += _parse_match_section(s['matchBinaries'])
     return output
 
+def release_notes_changes_section():
+    return textwrap.dedent("""
+    ## Changes
+    The following packages have been upgraded, to address the mentioned CVEs:
+    **todo release facilitator: fill this in**
+    """)
 
-def create_github_release_notes(gardenlinux_version, commitish):
-    commitish_short=commitish[:8]
-    output = ""
-  
-    manifests = download_all_singles(gardenlinux_version, commitish_short)
-
-    output += generate_release_note_image_ids(manifests)
-
-    output += "\n"
+def release_notes_software_components_section(package_list):
+    output = "\n"
     output += "## Software Component Versions\n"
     output += "```"
     output += "\n"
+    packages_regex = re.compile(r'^linux-image-amd64$|^systemd$|^containerd$|^runc$|^curl$|^openssl$|^openssh-server$|^libc-bin$')
+    for entry in package_list.values():
+        if packages_regex.match(entry.deb_source):
+            output += f'{entry!r}\n'
+    output += "```"
+    output += "\n\n"
+    return output
+
+def release_notes_compare_package_versions_section(gardenlinux_version, package_list):
+    output = ""
+    version_components = gardenlinux_version.split('.')
+    # Assumes we always have version numbers like 1443.2
+    if (len(version_components) == 2):
+        try:
+            major = int(version_components[0])
+            patch = int(version_components[1])
+
+            if patch > 0:
+                previous_version = f"{major}.{patch - 1}"
+
+                output += f"## Changes in Package Versions Compared to {previous_version}\n"
+                output += "```diff\n"
+                output += subprocess.check_output(['/bin/bash','./hack/compare-apt-repo-versions.sh', previous_version, gardenlinux_version]).decode("utf-8")
+                output += "```\n\n"
+            elif patch == 0:
+                output += f"## Full List of Packages in Garden Linux version {major}\n"
```

Comment:
I won't like to see a list of all packages in Garden Linux Release notes. I think it won't be consistent since every flavor has slightly different packages included. This becomes misleading.

Reply:
Reasoning for the requirement is here, why are we debating this when the implementation is done? https://github.com/gardenlinux/gardenlinux/issues/2295#issuecomment-2309512252

Reply:
Jan, right now the "release note" contains information about all releases - it is similar with the CVEs for example - they are not grouped per image or flavour, but for the release. As of today, I'd prefer we keep this behaviour consistent and later improve it as you proposed, but that would require further thinking how to actually re-organize the release content. 
I'd prefer we keep the proposal as-is done here.